src/components/DocumentationLayout.vue

```diff
@@ -245,7 +245,7 @@ export default {
 }
 
 .documentation-layout {
-  --delay: 1s;
+  --delay: $nav-time-transition;
```

Comment:
Can we rename this to better reflect what the delay is for? Or maybe just have the variable be `--nav-time-transition` or `--nav-transition-duration`?

Reply:
Sounds good! fixed